ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -75,7 +75,8 @@
     private static final By EYE_REPORT = By.linkText("Eye Report");
     private static final By TELEPHONE_NUMBER_TEXT = By.cssSelector("#contactInfoContent div span.left-margin");
     private static final By RECENT_VISITS = By.cssSelector("visitbyencountertype > ul > li:nth-child(1) > a");
-    private static final By PATIENT_GIVENNAME = By.cssSelector("#content div span.PersonName-givenName");
+    private static final By PATIENT_GIVENNAME = By.cssSelector("#content div span.PersonName-givenName");   
```

Comment:
Is the change necessary at code line `LN-78` ?

Reply:
There is no change here thanks